app/entities/user.go

```diff
@@ -8,12 +8,12 @@ package entities
 // Entities like User are the least likely to change
 // when something external changes.
 type User struct {
-	ID        string
-	Email     string
-	FirstName string
-	LastName  string
-	Password  string
-	CreatedAt string
+	ID        string `json:"id"`
+	Email     string `json:"email"`
+	FirstName string `json:"first-name"`
+	LastName  string `json:"last-name"`
+	Password  string `json:"password"`
+	CreatedAt string `json:"created-at"`
```

Comment:
Do we really need to return created-at? I mean we could use json tag with "-", to not display it through some GET request. IMO it's our internal field.

Reply:
@lifehou5e Maybe we will create some kind of private User without a password and other sensitive fields for some reason in the following iterations?

Reply:
@delveper may be, but we are talking about json tags. Where will they use other than in http response? We don't need it (field "CreatedAt") private, we just don't need to share it through json response.

Reply:
This  struct is for all our program, not only for request

Reply:
And where do we need json tags in our program other than in response to http request?

Reply:
i just  get this struct from our DB struck. It must be the same.

Reply:
> And where do we need json tags in our program other than in response to http request?

where have you been, when we tack about it in telegram? )))

Reply:
JSON tags has nothing with sameness, imo)

Reply:
oh, maybe i don`t understand you. Also we have no reason to display password in requests? Its better to make correct export form DB without "created" and "passwrd"...

Reply:
Well, i am waiting for someone who have better knowledge. Cuz right now the only way we interact with json tags for me is through http request/response and if so we don't need to show our internal data. Maybe password too.